.travis.yml

```diff
@@ -10,6 +10,8 @@ branches:
 language: php
 
 php:
+  - '5.6'
+  - '7.0'
```

Comment:
Isn't this supposed to be managed by `dev-kit`?

Reply:
It very much is, and will be ignored / overwritten. Can't make this PR without ensuring it actually works, right?

Reply:
Oh you'll create the PR on dev-kit after you have tested that no test fail, right?

Reply:
Yeah, not sure what the plan is regarding the merge order though :/

Reply:
https://github.com/sonata-project/dev-kit/issues/315#issuecomment-339435597